config/routes.rb

```diff
@@ -303,6 +303,7 @@
         show
         tl_chooser
         wait_for_task
+        show_list
```

Comment:
@abonas This wasnt a must. However every other containers entity implements a ```show_list``` action and a ```show ``` action. This allows us to redirect to an "all containers" screen the same way we do for all others.  

without this, I would have to add an ```if entity == :container``` condition [here]
(https://github.com/ManageIQ/manageiq/pull/5142/files#diff-b42f7ac43b0b8ad4c9ef885aeb863d10R119)